CMakeLists.txt

```diff
@@ -36,11 +36,7 @@ endif()
 ## Pull in helper macros for working with leatherman libraries
 include(leatherman)
 
-if(LEATHERMAN_USE_LOCALES)
-    set(BOOST_COMPONENTS locale)
-else()
-    set(BOOST_COMPONENTS regex)
```

Comment:
I think locale should still be left out if `LEATHERMAN_USE_LOCALES` is disabled. I think some platforms may not have it available.

It seems reasonable to always add regex though.